addons/message_paid_research/currencyCheck.js

```diff
@@ -0,0 +1,14 @@
+(function(api, condition) {
+function computeCondition() {
+  if (['USD', 'CAD'].includes(api.subscriptionData.planCurrency)) {
```

Comment:
Acceptance criteria doesn't mention CAD - should it be included here?

Reply:
It's OK to include Canadian folks, but we don't actually pricing in Canadian dollars, so wouldn't they all be rolling into USD anyway and CAD be an empty set?